app/index.html

```diff
@@ -1,42 +1,46 @@
-  <link rel="stylesheet" href="css/app.css"/>
-</head>
-<body ng-controller="MainCtrl" class="zipcode-{{applied()}}">
-  <!-- View placeholder -->
-  <div ng-view></div>
-  <div class="loading-indicator">
-    <div class='text'> Loading ... </div>
-  </div>
-  <a href="https://github.com/shidhincr/lookaround"><img style="position: absolute; top: 0; right: 0; border: 0; z-index: 10000" src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png" alt="Fork me on GitHub"></a>
-  <!-- In production use:
-  <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.0.7/angular.min.js"></script>
-  -->
-  <script src="lib/angular/angular.js"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/0.4.0/ui-bootstrap.min.js"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/0.4.0/ui-bootstrap-tpls.min.js"></script>
-  <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=places&key=AIzaSyCOUo69jud8ZVGG4sa7ijZ3y6XL_XNmkXg"></script>
-  <script src="js/app.js"></script>
-  <script src="js/services.js"></script>
-  <script src="js/controllers.js"></script>
-  <script src="js/filters.js"></script>
-  <script src="js/directives.js"></script>
-  <script src="js/utils.js"></script>
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+  <head>
+    <meta charset="utf-8">
+    <title>LookAround - Angular JS Learning Example App</title>
 
-    ga('create', 'UA-10379928-16', 'lookaround.herokuapp.com');
-    ga('send', 'pageview');
-  </script>
-</body>
+    <!-- need to change these to local -->
+    <link href="css/normalize.css" rel="stylesheet" type="text/css" />
+    <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" />
+    <link href="css/bootstrap-responsive.min.css" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="css/app.css"/>
+  </head>
+
+  <body ng-controller="MainCtrl" class="zipcode-{{applied()}}">
+    <!-- View placeholder -->
+    <div ng-view></div>
+
+    <div class="loading-indicator">
+      <div class='text'> Loading ... </div>
+    </div>
+
+    <a href="https://github.com/shidhincr/lookaround">
+      <img style="position: absolute; top: 0; right: 0; border: 0; z-index: 10000" src="img/forkme_right_red_aa0000.png" alt="Fork me on GitHub">
+    </a>
+
+    <script src="lib/angular/angular.js"></script>
+    <script src="lib/ui-bootstrap.min.js"></script>
+    <script src="lib/ui-bootstrap-tpls.min.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=places&sensor=false&key=AIzaSyCOUo69jud8ZVGG4sa7ijZ3y6XL_XNmkXg"></script>
```

Comment:
@sahilbhatia your changes for removing the `sensor=false` got overwritten here. Could you please update this ?